modules/seqexec/model/shared/src/main/scala/seqexec/model/Conditions.scala

```diff
@@ -45,12 +53,10 @@ object Conditions {
     )
 
   val Default: Conditions =
-    Worst // Taken from ODB
+    Unknown // Taken from ODB
 
   implicit val equalConditions: Eq[Conditions] =
     Eq.by(x => (x.cc, x.iq, x.sb, x.wv))
 
-  implicit val showConditions: Show[Conditions] =
```

Comment:
This wasn't being used I presume?

Reply:
nope, we were using `Show` for the conditions components though